extendeddatasquare.go

```diff
@@ -154,25 +154,19 @@ func (eds *ExtendedDataSquare) erasureExtendSquare(codec Codec) error {
 }
 
 func (eds *ExtendedDataSquare) erasureExtendRow(codec Codec, i uint) error {
-	var shares [][]byte
-	var err error
-
-	shares, err = codec.Encode(eds.rowSlice(i, 0, eds.originalDataWidth))
+	parityShares, err := codec.Encode(eds.rowSlice(i, 0, eds.originalDataWidth))
 	if err != nil {
 		return err
 	}
-	return eds.setRowSlice(i, eds.originalDataWidth, shares[len(shares)-int(eds.originalDataWidth):])
+	return eds.setRowSlice(i, eds.originalDataWidth, parityShares)
```

Comment:
is parity shares the entire of row of shares, but we're only interested in the later half? or is this actually only the parity shares? not sure if it matters too much, but if it is the former, should we change the name to parity?

Reply:
`codec.Encode` returns just the parity shares so we don't need to index into it.